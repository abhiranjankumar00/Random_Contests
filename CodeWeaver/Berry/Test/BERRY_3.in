18
42 858
6053 33008 15221 4519 770 909 12999 18038 19333 20335 20512 6064 22343 15388 12155 2486 20529 26284 26916 17652 10511 29320 24428 31939 30950 18290 5281 15758 8328 28870 7485 26164 28261 33279 33374 20654 13476 9340 10358 28146 1395 12933 
301 66
22807 2459 905 30723 9657 9559 19881 23265 28063 24422 16507 7346 24987 15967 15623 31718 25010 4165 20246 13240 24701 20571 19373 12457 5583 7993 14548 26925 17561 33056 25243 11940 6154 22155 16735 2699 23546 17158 5450 31732 5256 30275 23321 10547 14594 2923 32290 23003 5638 29362 27541 19689 6887 6446 27394 16190 5714 17129 24603 13907 9778 19253 22972 25384 24272 16191 4648 12669 27967 1767 17558 22341 14583 4047 31724 30808 18624 11954 16074 12904 28370 11000 15562 9526 32826 10375 24838 19205 7039 25563 28123 13647 17030 29785 1517 33208 19379 2272 18027 6976 28241 21154 22165 7100 28212 4658 4219 24967 10908 31453 9991 11637 5213 24638 27965 23816 20285 25101 8898 5351 670 2616 27035 28555 7517 4405 18744 1084 4345 15510 25265 23100 18424 3038 14927 7234 23430 10131 14926 8233 19552 18068 21217 5866 17136 32173 23415 30920 6841 11625 6461 33226 15606 33319 3812 21102 3710 14960 31140 2204 29931 10120 8065 25817 14075 7235 21197 20413 15415 15898 20206 21415 28626 27774 21089 24239 19155 12909 6655 24796 18807 30274 8575 6808 20432 23505 19773 12128 16819 3251 24889 20672 20945 23263 26979 5503 19492 29050 30486 20759 24461 2033 22498 32615 10939 19268 10638 6316 4174 29093 28509 25006 5692 11329 3217 3127 21620 27486 18478 15305 14018 10406 15872 17040 5619 11202 24799 10902 27705 32043 29234 24374 10449 31549 4786 2771 26338 17806 20593 31946 2985 29309 22042 11172 4051 17166 8058 10129 30762 18677 22335 4370 16756 13040 16896 26613 17304 13146 23166 18054 20483 13566 810 23812 3914 21033 28903 16722 25105 16899 13072 26351 18731 12318 31933 21288 6403 12880 4930 24972 31802 20670 1529 11425 2449 21562 4930 28373 28938 20175 24263 28599 1098 4761 23813 20839 7746 2778 6152 21067 11790 
443 497
12661 22276 26909 27056 17870 20641 26771 30127 5362 21266 13803 8930 5091 28639 2785 14635 19404 10522 18575 8008 10819 4242 27094 5031 9931 26441 29951 26491 24989 20776 12450 28687 8104 28914 8135 8794 18390 32887 2380 3285 17809 14602 3719 9097 29830 8244 5419 4716 31144 3242 2160 2541 11640 30589 13678 14798 31620 28049 18122 28210 23342 13648 9109 759 17839 31903 13681 8517 19804 8977 23030 31591 14887 29787 852 31450 20328 26377 19513 5284 7902 13167 757 14311 3312 22241 28636 2742 27193 22311 23275 26919 17988 23083 5295 23210 24468 26101 1635 29081 21736 23769 3417 18704 31677 2543 9627 5374 32787 20932 10193 30844 12660 23375 7781 5726 8499 17703 30182 4104 30194 15091 3242 26244 6246 32545 26040 29137 7519 15579 740 25213 4354 13878 10812 33142 1881 8877 6819 8733 5647 20943 29628 31388 7028 26164 21770 22137 30407 15541 30805 22380 5557 21468 19279 14446 31517 18330 19575 18195 8982 13624 12094 1817 26675 7056 2920 13377 19974 2941 7094 5725 21730 15504 20418 23467 15077 23045 13610 28001 20022 7157 31728 29321 13452 32973 6689 29967 19440 15102 21173 23833 2243 9974 9143 13974 29954 23888 24486 29978 13513 13639 30234 10729 13334 19313 12202 24658 25040 24746 24846 5182 7176 9549 22913 30472 12670 17579 4201 30351 7271 17589 22074 14986 23149 3580 3224 3100 1528 12252 10684 23465 5292 13555 2840 22759 24371 7128 23358 20792 22076 18442 3838 18487 2243 32551 15822 32400 24467 18015 22896 23318 12810 13351 2254 27151 6173 2610 10793 26322 7579 11165 16498 917 15030 17258 20472 25472 13759 12079 28228 31494 21041 33295 2369 19060 28913 13290 10657 9930 1492 20689 13435 1227 1634 15549 28729 31312 13786 24448 20657 27105 29302 19707 20114 3206 26607 15762 19970 19931 19268 27357 22661 11875 952 9376 11809 8222 25881 12798 29120 4446 24392 21225 29909 6614 18166 30173 26178 32244 7084 30158 26746 31857 28430 29791 21218 6694 20832 9300 8830 24609 26441 6218 24009 16376 3287 27045 28644 4701 10692 19403 16115 25587 6904 24464 33221 1120 21838 23280 6323 2980 9627 22788 28106 18857 12781 26159 8236 21530 8208 7321 32435 9849 1367 25822 5592 2156 920 12039 21715 11864 14201 14265 3588 18481 23530 11692 22162 29329 18163 19295 2386 15879 9136 14602 28906 16561 3482 19157 23862 26125 28935 28487 10218 28940 21749 25089 27769 25704 29187 30380 30518 17720 15040 4144 12217 27586 14632 6182 645 22160 25964 2705 3560 13254 23026 27080 5666 5298 2205 11959 1225 32677 21838 2375 30571 21505 14763 1307 8566 13460 22047 26237 1365 27684 5153 21400 18652 32843 15802 7655 10470 
478 660
11841 16365 20540 3976 32874 25383 5561 15572 23935 33343 26028 21642 20972 31618 23166 10113 19201 10132 18581 23959 8692 18530 16759 31813 22565 25689 21043 15223 26790 25046 33021 7322 21243 30429 19992 30710 17182 16653 15353 29758 31431 6611 8873 7269 24553 31455 21099 1972 21632 1748 18847 18458 15242 2628 33254 4803 20307 2267 6279 22251 11510 10970 22033 14069 14625 7879 22850 6071 16920 17251 26920 14873 765 11741 12934 4830 27726 24645 29817 21697 21935 17475 24667 24772 31978 15881 12520 4293 30260 27526 15519 4178 29106 25565 1899 14270 31291 2837 6019 7540 4448 19986 4495 20876 19625 23236 24588 30620 22562 10398 27121 13009 33302 25930 7315 6206 13339 6555 19654 7027 32078 6851 13125 10578 4576 7110 2132 25886 25554 29486 675 15317 14588 26881 15734 16832 25172 8432 18947 3656 30829 29905 28699 2571 16702 21327 25101 25391 8997 30432 26568 7158 18732 2296 17727 26515 23690 1405 24594 18749 30573 29306 26730 10625 13558 16407 2340 21980 645 25801 25395 7817 5688 2502 22591 1688 11950 23484 26770 14912 32785 21442 10293 16246 29267 3215 3926 5310 5303 30697 10733 12866 27284 2145 27296 8481 30038 9022 9443 7433 8133 28343 29283 29704 28298 20581 1957 6303 6638 12279 15338 6681 29926 21032 18519 30490 17741 24558 33362 32379 29040 28148 5423 12885 9935 6219 32524 24924 24098 1629 21753 27247 24731 27426 4701 24173 12907 21910 30613 29769 1671 25962 15325 865 8108 14329 5748 30731 17056 8575 25339 30321 7612 1669 25886 8096 18855 27040 28535 11483 18276 33115 29103 21813 8079 27250 12775 12639 1763 19009 13654 4542 11264 30075 7219 31424 4715 28447 29909 15217 7895 21416 18969 21065 14499 1157 28932 5192 9379 10184 22953 19014 32035 20514 26445 14468 27433 24847 5066 17542 1275 846 25137 11557 30177 15764 25779 2272 21292 1409 22978 32371 17015 23763 9331 27563 21805 20186 9255 28280 15692 25132 12138 1092 16232 21145 13851 13187 1522 10081 20174 29579 14778 31141 13166 18086 947 32540 28967 14950 23542 11527 20354 16953 18954 1895 13010 8253 31331 28781 32208 20566 19752 3122 13954 10621 16954 3240 29281 10587 26225 31185 19898 7702 9769 18990 29269 8811 10493 24443 29013 8189 6866 15257 29791 19436 1560 33228 2558 20906 9036 6182 4422 8202 16833 32067 2232 3226 11681 9783 17510 16691 14126 30917 20312 10190 16206 13778 31789 18585 27975 26648 26885 20043 29127 11039 31391 9396 31527 24169 4799 26008 16112 6381 1491 1981 15519 28872 5077 1091 7589 27438 26739 26164 22194 10406 26222 30812 28211 8567 16960 1371 3098 1629 12289 24056 14045 6529 10379 6584 25025 5032 2525 17840 12376 27051 5821 30103 4936 1575 6562 27423 8783 19317 11984 18309 22404 29149 31173 31591 4064 5638 18900 26839 29948 2993 29200 27565 13340 18599 22635 15393 9837 31728 32799 24473 20107 9542 
21 355
7103 20294 16586 12260 8151 16294 23285 31685 22608 32813 8400 17445 8344 30043 16488 11314 18229 19320 11644 12025 6162 
13 539
7802 30855 24854 27060 21930 23118 9467 25745 2728 24267 28766 11809 2071 
186 900
1298 28082 20884 18330 20410 19251 26089 21589 23447 25518 29059 5794 14494 26387 1319 14981 1998 17207 9440 2854 8640 32796 17212 20786 2944 12659 31927 28799 11675 8543 10104 2287 33145 6268 2875 7830 10477 26400 21903 3359 9227 5089 7745 19051 6458 21335 4120 15484 25482 11555 2498 9874 23367 29231 19215 29075 3278 26680 33373 21424 20895 20106 20052 6982 32692 8203 2003 8450 8977 13947 26665 9545 30109 13050 24457 7383 2352 20427 12216 12454 22509 32599 7945 29969 11986 31861 22411 7225 9869 12264 18049 10152 7825 3881 4114 23860 28266 22632 27458 806 16717 20429 31265 10140 20468 25091 29250 30508 27817 20471 10607 26314 5536 18397 23195 13548 33161 3957 14104 1516 23142 9100 6155 8109 6810 28869 25628 2177 16451 2457 32114 3402 19146 32157 19422 29408 14009 13827 14166 22195 27596 31588 9894 690 20810 2237 22114 14443 31893 20407 7199 14195 11776 21035 27106 776 4066 13541 9689 10997 27625 28584 10825 7094 19120 15987 33224 14813 12691 14339 13228 19636 29378 11673 13507 31780 17300 14333 20549 26069 3358 5571 7430 3479 11013 16077 
390 646
11521 3216 20753 5297 9815 27649 15004 33201 7857 4134 17623 9287 858 19499 25866 4140 32375 12722 30238 17712 7913 10533 16295 10795 14196 25675 28194 13367 14624 24346 24698 3612 3295 25293 28370 21516 12401 16182 3710 14225 6789 11860 30546 8131 10785 5672 11808 6590 9539 7365 6268 11603 2761 17063 32194 27134 15246 30533 6491 16408 16086 23529 8763 31310 9708 10432 27566 27474 21257 26669 853 9168 10104 31843 28779 17572 23800 9011 30797 16714 3682 27682 3130 9878 13195 21695 17495 14757 5228 32564 3062 787 30592 14106 25367 6766 3950 14482 22831 17899 27624 28049 9513 16984 29139 5499 32427 18258 7848 2804 7322 7096 6938 7155 31968 20064 24825 10263 22893 26376 30806 31039 28283 9103 4072 2645 30231 5477 29870 12394 31965 13120 11478 23048 29699 13072 14814 22723 9369 16438 17913 26898 26638 25072 16564 27741 18935 9752 11993 2391 30476 25615 12691 9870 7539 11252 2260 29505 8770 13849 5364 29297 20543 26295 7067 15418 25578 14816 21472 26980 6333 11190 32178 23862 7090 28891 2167 24970 1798 4904 3316 18727 19260 18082 7789 3146 10598 2729 14093 4998 8864 27507 17288 9422 32989 30649 4606 9937 23782 16048 10275 11910 17823 27195 17565 2823 27788 18783 4013 17314 31637 24311 16146 15405 30726 18601 12586 29595 22234 4632 17243 4301 9785 18523 26545 12714 28735 25007 15816 12402 15103 19816 13037 21327 18888 16430 28032 6702 935 4197 1948 7884 9004 14061 23803 15124 4977 18444 28691 27048 7769 30888 10885 5310 20502 954 31928 18221 13294 1023 8385 29639 20030 5617 1159 18154 30655 22005 7972 13906 32264 3798 13710 17201 11604 11378 21110 8834 10118 18099 30064 16282 11370 22721 15701 19712 14142 30838 4581 22398 5489 8723 4273 3960 23210 27697 16697 15485 22028 14330 21585 3868 27833 12696 29385 17657 32148 13550 24396 21280 12876 27270 28553 2712 24982 12491 8337 27452 26297 9593 19072 6342 4688 25425 14176 9773 6219 18449 8522 11057 7992 22179 23502 22579 21697 2453 11414 31819 11125 10697 5957 12693 9675 32110 8357 9163 21091 25390 19714 9497 23952 11689 24017 1324 23572 21283 4758 24205 13098 17785 28932 20013 16842 1626 22629 18753 12072 19379 25660 1359 638 18539 25129 9865 26635 27985 24334 10331 18267 13907 3952 13376 3786 22512 5920 23257 22596 24816 29722 25408 
449 692
9298 4269 9370 24650 32646 16604 7659 26870 20700 12037 16431 33335 12861 31680 7846 21269 19481 5386 31087 15622 21664 2211 22104 24943 3893 21940 7157 8296 24607 33205 25327 29888 18562 29838 29087 20800 24206 24572 5595 25396 5623 13879 20752 24104 27813 29650 2705 13552 20396 16610 15407 25579 19582 11618 19722 12424 25184 3286 3751 33288 2074 13615 4162 14214 14835 9455 15461 7989 9180 21645 29881 29418 15340 14107 8221 32218 24844 11814 24309 20388 20980 29918 22980 8338 4182 18038 24062 19227 1112 17358 15639 32429 29768 23632 8672 4010 9355 15896 20499 2225 13271 4439 33324 24862 1955 761 22907 31956 23771 25221 5240 22842 20695 22341 32598 2947 30350 28840 21211 29747 19463 5604 30062 19422 9233 6367 21106 27955 22580 28752 15314 32826 22247 23841 2525 27354 25091 32537 27989 10910 4472 7351 10178 31673 28223 8801 7692 28999 14736 7358 31049 25301 13396 12896 18900 21284 31453 24742 24735 13354 18645 32736 21194 11354 14708 21173 23412 16677 22518 9410 4049 2753 5197 15492 6524 21801 28691 9336 27407 21093 30658 3968 8946 16907 23591 23689 4208 12806 16404 32107 3856 15510 3670 17680 11298 26520 27622 24793 19096 2312 6756 33085 14615 21700 24256 30132 28238 30610 16443 16339 24455 25445 13198 25016 9595 12508 1640 27560 25451 14860 7966 3319 9324 21119 25758 11304 8963 16508 25254 19030 12017 23433 32307 13779 9552 29743 20027 20494 11999 26877 27806 27322 14491 3280 21339 32812 23183 16977 6655 19205 8428 28862 10084 33042 27484 13806 11159 13680 22164 31119 30234 9716 25640 17843 31820 30665 21668 2187 9253 10075 26584 9073 10022 12148 30579 27436 8592 11173 8148 32112 8661 31208 27420 4005 19532 8067 12936 6889 26408 30000 10750 2392 25259 24186 18917 18571 12695 4202 2874 25950 1510 11107 10720 5367 17850 11339 24888 20228 19607 30955 22258 23875 19664 2969 10275 1096 25919 33300 19607 10619 16185 5039 17512 6752 18625 12978 17359 22811 24316 23805 20795 4659 31974 24558 2464 8086 16962 4997 32530 15581 26327 20678 26002 20690 28121 10079 33320 28719 11686 5771 4146 32169 32691 19766 4347 2861 11461 13666 14735 24682 26980 20169 29390 19087 30348 7364 8128 6486 15015 11115 7720 27969 26296 28670 13660 15139 31705 2180 26451 10467 17639 7857 22528 10109 6268 29197 16094 28233 30493 19795 5837 15232 22965 27415 32796 32252 17474 3500 2279 19519 28879 20037 20387 19639 9890 26458 13303 29505 4534 32334 2509 14315 11884 12366 682 11006 7549 26236 16698 2620 29747 13645 19248 15206 23769 19247 19789 20481 7024 12912 26045 30298 19107 27227 26794 31688 25853 7076 21260 17457 28250 27654 24510 23247 20891 13610 29399 32820 18078 
406 321
20279 22924 21614 31768 19040 9507 17334 7257 29320 6183 5500 24278 23276 18862 21003 33268 33382 20218 17457 17826 2114 27900 29867 18675 9038 792 31198 27404 3888 4904 28616 13121 25596 2755 11317 1147 3078 12735 822 8157 19445 25076 12239 15015 4377 27715 13552 11098 31231 20184 19110 29213 19778 8363 11155 8287 21155 22093 22172 25534 16988 17377 6369 14375 16050 32872 29682 1343 3018 3651 13376 11010 31165 27890 19409 16810 13550 30754 17720 4307 28828 14311 25285 33244 8115 1606 15865 4496 12264 11766 31548 1286 31492 26215 26066 26053 25216 20740 2060 2348 18771 23478 5051 10002 24584 11433 12010 17321 30483 31570 30925 28840 19301 31409 23284 10665 33086 14547 31428 20431 16707 6044 24360 2221 11216 17091 7812 1123 5775 18206 10475 28426 18506 12064 657 4435 9981 18592 643 25706 3120 30683 27029 7457 13187 15293 26012 29258 30603 18268 21726 8804 20700 21702 24007 31867 19303 5372 11998 28745 26119 24894 31479 27041 4182 4489 1449 31502 15045 7263 21506 7095 11500 25542 21882 31877 5662 21395 6438 14472 3137 13544 25390 25751 27868 23292 13953 16951 21618 3354 9927 6591 8207 20973 1688 32697 11552 32762 17899 2442 19335 14190 5329 13654 5615 10522 7779 2334 9309 20156 28664 11563 19720 30848 24732 1456 26475 21446 3262 4011 17816 20262 14889 1292 22479 32689 14390 3097 11855 27514 4159 30225 19820 23894 6742 20751 2018 9695 18999 20786 21353 28095 18556 21755 19338 32043 11280 2294 22192 7341 26598 28317 8578 29479 16619 19065 25712 1301 24633 15754 10124 14610 21901 3561 30876 1438 7675 25260 18285 11294 736 30474 23658 9508 25881 2240 7532 8689 18546 1458 5332 18232 13153 15574 28376 1317 14069 5710 24858 12446 13065 5844 12415 16841 10941 12891 18750 23502 15655 12787 7479 33246 5701 12526 15495 7540 27702 30017 23641 14810 23722 10029 16382 6384 16024 11294 14401 1321 15407 10451 11508 1238 18702 28351 25807 21159 18615 22703 32635 16238 31041 23214 21993 24871 30286 18725 13544 13110 26685 6983 9719 3055 13217 11436 14935 2907 15601 16956 3173 26425 5249 22095 27431 14404 7856 28691 25637 11444 3171 2605 8058 17415 14259 19619 16938 30721 14563 7673 19126 30450 18325 20378 7150 21303 15421 27002 10759 19459 17877 22778 32237 4960 10195 1804 730 13680 5208 6551 21032 1273 30378 25629 29028 12576 16868 19355 13987 26741 6124 8624 18201 11568 23718 15106 25575 16058 
305 821
17630 31029 15472 21677 7710 9089 26019 33293 14160 7990 6883 28268 15892 22145 1149 11378 6086 33120 33346 17724 28509 3068 18570 29009 23187 14136 20089 24024 23111 10122 29057 21000 14879 22026 29224 18655 4977 6088 11788 21271 30564 12131 15140 21846 1106 7383 18504 30373 27249 8087 26356 21283 9139 7900 8777 18170 25322 27708 1876 26003 32304 5246 659 28934 22662 12973 14005 26069 3771 31083 12450 28377 27536 26248 7355 21884 26393 21715 24931 16937 13670 24624 17086 24833 10339 20459 29624 16080 26877 31564 14410 29247 19334 23660 31993 25931 17243 17734 1951 21237 14788 27409 9581 19771 19483 15310 31281 24392 13617 12829 21717 8168 26093 6996 11560 18494 4406 978 12002 8889 12051 15761 30178 26180 15121 13563 24505 20925 12774 2047 5740 7081 4691 4963 14454 2598 25935 27228 16909 18208 4449 27740 7078 17608 32078 27947 29115 17986 17908 9017 14796 24734 2934 20531 2212 2058 6553 22484 7645 32070 28526 14590 18598 12120 27186 7433 2500 25984 6002 29019 5847 17774 29298 16184 8927 2483 4304 13788 4094 31862 33381 6855 28288 2664 26732 5209 27928 33274 2022 670 7788 21197 23357 23147 14292 24937 26407 13994 11615 17203 22221 5106 20248 30361 9005 27227 19335 27536 18876 6544 15226 28736 22096 23333 6917 22013 17470 12075 21447 25210 31423 30466 15471 24889 26777 12911 12671 26950 763 4846 11088 11045 11176 22858 30038 23305 16983 12650 27302 2887 25946 32878 23477 2825 9599 22779 33123 7495 12596 2514 8504 7155 15086 31400 18157 23412 7883 19250 14126 25174 2934 26751 32670 18752 12875 3210 18460 22310 17083 23568 18757 11711 13554 5791 3129 6714 10168 3312 22434 24109 21909 20854 28147 1230 25913 22327 33006 6245 6426 29403 32361 19457 8799 29186 6737 28010 6909 15579 14831 13144 27424 13811 9322 15330 7628 
159 944
7310 20281 26820 29612 18379 14998 27563 30526 1665 22385 27241 23128 8571 4852 1744 15250 28857 2479 15744 28194 24434 2957 10759 15717 29017 7175 22093 3771 11552 17047 6448 21912 4154 16240 18539 3544 28646 30828 16828 12371 27497 12870 3910 21938 21402 27145 11060 31617 11172 25288 9887 8069 9992 15248 14257 9551 18657 8603 29451 14865 2156 29382 13413 10929 7879 25669 31348 24604 25081 33174 14865 30173 20787 6067 27799 7918 13910 24716 23949 19661 32948 692 31621 19667 15643 14683 12355 20418 20240 14460 1058 3997 20312 8535 4276 1980 13245 6946 1705 29471 9230 25596 19186 1283 8296 28194 28454 4019 23732 23909 2340 3489 30065 30940 11770 16565 25012 25814 5125 23525 18028 14610 8282 27398 3819 18227 25445 27873 28322 9582 18640 21386 12332 20870 32422 24301 30095 32762 22457 1450 19902 27517 11585 24611 17250 14502 965 18898 18649 16873 30527 11489 9599 24518 12923 11252 18477 33088 30144 
205 168
8960 10385 28850 4612 29788 8220 29559 18628 8496 31651 15129 25210 1147 14778 23851 10875 22543 6523 5808 15060 6805 30844 33336 27101 5841 19710 31052 13841 28711 4199 9484 9244 22622 11763 33303 17206 21157 18854 23467 19376 10195 29573 33294 3020 28659 6917 16428 5842 21760 22212 21633 27232 29421 21623 28985 32258 30952 1315 1942 28930 28640 22929 25235 19074 19013 8956 21703 32711 8463 27660 32139 13528 16897 24072 26975 27464 29487 1607 4490 4944 4030 4071 32513 13089 25067 17858 5258 6263 24142 28830 17441 18546 9312 26020 13211 8610 26684 22804 5257 25696 10658 18742 13151 5477 23617 5524 33238 28309 25910 5734 8334 21020 4560 18510 11256 29628 27542 24645 20595 11365 3467 23509 27061 11271 24240 24970 12653 32164 21963 1443 16678 4521 10855 18188 27503 17022 5916 22673 12345 1884 4368 2636 20608 18783 6334 26463 17710 3804 2038 23529 7484 28916 33061 30514 1818 3945 25559 7721 2385 30385 27258 16913 11879 11337 21861 28535 10242 21836 18201 18791 11601 11076 13766 17631 6312 13120 5649 14468 2154 27431 8980 5268 1836 24907 26534 22300 32535 27201 27869 4203 31886 26865 12754 3014 1041 23785 20644 28765 32610 10999 29888 28286 1510 21494 1631 
134 135
1101 3560 12112 27428 1910 29509 7415 2607 11916 15507 9110 29691 2166 14391 9931 20790 24686 29835 27482 17450 15855 20528 13869 7080 7125 25766 23705 12862 6848 32537 10723 19888 12985 7038 15875 4774 10264 5599 18855 24739 18340 1523 19323 10806 11773 30962 12968 7540 7735 32167 2714 12453 11208 14023 5127 26430 15268 26497 28461 25360 8358 11862 15195 5566 22322 25131 20402 9844 9241 17821 28973 24534 6844 1172 10719 14903 20085 12315 9596 22028 13106 11375 14107 31392 29027 17749 27706 27913 12567 23729 20739 13146 8289 26453 18894 6170 14593 11287 9702 33269 2368 31014 20368 29618 1052 17310 15689 18533 7333 25945 2128 32742 12455 27913 19556 12161 1673 13443 13519 27375 25205 12638 23203 14725 30394 13245 22596 27564 13166 1897 13862 25745 14679 11043 
142 750
11294 29708 4664 14402 23848 8023 9301 22571 17386 15326 28573 23038 16562 14082 16633 22477 17114 18952 5807 6294 28666 21622 8503 24005 19292 13760 17994 10747 29104 27172 23099 25870 15617 28371 906 33105 25832 12625 31927 1557 28416 3614 10324 9071 921 8854 11102 13116 31409 30230 33229 19299 13600 15968 16746 17754 5650 17217 32481 24300 12474 9585 2327 22585 32325 3258 28349 13054 12301 33116 33318 30486 29200 29873 20309 8257 12704 15629 14427 6437 15452 2280 11232 26514 19294 3391 10751 28170 20150 29495 2903 26241 3351 18692 15966 31329 27601 31241 28409 7121 29439 4672 19530 1079 25090 20701 24777 31739 26670 20014 12439 20367 21954 14534 4623 5548 22814 28122 29853 10974 7216 28907 2515 1602 22064 1701 1209 19342 9613 30472 25445 15955 17959 15045 30412 31359 30654 6123 7384 5670 10410 29122 
1 177
4226 
50 955
20471 30082 12856 26823 24826 16109 17116 18792 31653 23678 14264 16235 25630 27344 16329 28513 22526 28458 20747 9707 9651 8532 6903 4140 21283 1961 33353 22654 6033 6510 2566 31588 11461 23394 19902 26817 15207 10902 28942 17735 26285 10623 10346 24625 15158 842 16153 12597 1333 31087 
159 415
29364 20945 28553 6821 11317 8140 26981 2028 16657 867 12121 24100 22383 16758 3287 11462 11123 9705 23873 25724 8883 6474 23295 14816 13291 14772 25808 9607 30449 31475 11613 9534 28907 4584 2324 8851 28863 12079 20019 8856 4043 19712 31523 8931 10967 5203 5896 18393 26439 7496 32918 16429 6293 3781 7974 33187 6655 24929 10761 17178 28797 18817 31260 22473 18678 3264 6556 20425 9907 18069 1596 33235 22036 24624 22901 25164 30594 16292 19230 19326 13988 29259 16956 6953 33238 22118 26651 18746 21975 2053 29086 23514 29121 5415 4819 23722 1855 15649 20682 13133 15098 16196 9640 2344 2388 31930 10999 6757 25433 29105 15936 6206 6130 18876 21468 32888 23900 4981 27655 3629 3801 22376 24118 10191 29350 5676 28331 28266 7212 29139 25158 21033 15994 1669 1182 20508 755 13927 21584 15672 14404 27841 18839 17056 28058 7265 27421 28451 19985 32058 9489 33285 20591 19793 5754 21250 3858 6362 18635 
