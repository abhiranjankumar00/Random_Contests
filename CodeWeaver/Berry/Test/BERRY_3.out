40 836
300 77781
441 15044
476 12546
19 931
8 286
184 3360
388 9835
447 11731
404 20873
303 6456
157 2913
203 21241
133 16018
140 3360
0 0
48 913
157 6433